Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -455,12 +473,15 @@ public void UpdateTripleValveState(float elapsedClockSeconds)
             var prevState = TripleValveState;
             var valveType = (Car as MSTSWagon).BrakeValve;
             bool disableGradient = !(Car.Train.LeadLocomotive is MSTSLocomotive) && Car.Train.TrainType != Orts.Simulation.Physics.Train.TRAINTYPE.STATIC;
+            // Small workaround to allow trains to more reliably go into emergency after uncoupling
+            bool emergencyTripped = (Car.Train.TrainType == Orts.Simulation.Physics.Train.TRAINTYPE.STATIC) ?
+                BrakeLine1PressurePSI <= 0.75f * EmergResPressurePSI * AuxCylVolumeRatio / (AuxCylVolumeRatio + 1) : Math.Max(-SmoothedBrakePipeChangePSIpS.SmoothedValue, 0) > EmergencyValveActuationRatePSIpS;
```

Comment:
I wonder if we shall use the pressure condition to trigger the emergency not only for static trains, but also if EmergencyValveActuationRate hasn't been defined in the .wag file. Currently, if EmergencyValveActuationRate==0, it is set to a default value at Initialize(). Switching to a pressure threshold by default would restore backwards compatibility for old stock.

Reply:
Some users have pointed out that emergency braking doesn't work properly on old content since 1.5, so backwards compatibility definitely needs work. While reinstating the pressure threshold would be the least disruptive way to fix old content, it would be more realistic if vent valves were added where missing. "Where missing" is pretty loaded though, I doubt a 20 year old .wag file will have enough information to accurately decide if it should have vent valves or not.

Reply:
The problem was that I initially set EmergencyValveApplicationRate to zero when I implemented the feature. Then I changed it to a nonzero value as default, due to complains. Neither of the solutions has been acceptable, so the only think that ensures that old, badly configured content still works is the pressure threshold I think.